runtime/frequency/src/lib.rs

```diff
@@ -521,6 +521,7 @@ impl pallet_collator_selection::Config for Runtime {
 parameter_types! {
 	pub const MaxMessagesPerBlock: u32 = 7000;
 	pub const MaxMessagePayloadSizeBytes: u32 = 1024 * 50; // 50K
+	pub const MaxDsnpOffchainPayloadSizeBytes: u32 = 1024 * 128;
```

Comment:
As per [DSNP spec](https://spec.dsnp.org/DSNP/BatchPublications.html#file-requirements).

Reply:
Drop Dsnp from it.